### PR TITLE
Ref docs

### LINKED ISSUE #341: Documentation: Reference docs WIP tracking issue
This issue is for tracking work still to be completed for rollout of the reference api docs.

### PR DESCRIPTION
**Pull request by peterschutt**
Closes #341

### CODE REVIEW

File: docs/reference/app.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
 # app
 
-::: starlite.app.DEFAULT_OPENAPI_CONFIG
-
 ::: starlite.app.Starlite
     options:
-        show_source: false
         members:
             - __init__
+            - __call__
```
Comment (Goldziher) [inline docs/reference/app.md:7]: this crashed for me when i tried it. working?

**Review by Goldziher — APPROVED** [review]

File: starlite/testing.py [hunk 2]
```diff
@@ -74,6 +74,8 @@ def url_encode(self, data: Dict[str, Any]) -> bytes:
 
 
 class TestClient(StarletteTestClient):
+    """Wraps."""
```
Comment (Goldziher) [inline starlite/testing.py:77]: this isnt a proper comment id say

File: docs/reference/app.md (hunk 1 above)
Comment (peterschutt) [reply docs/reference/app.md:7]: image

File: starlite/testing.py (hunk 2 above)
Comment (peterschutt) [reply starlite/testing.py:77]: ha I'd agree

**Pull request merged**